packages/components/toggle/src/toggle.test.ts

```diff
@@ -11,24 +11,6 @@ describe('OkuToggle', () => {
 </button>`)
   })
 
-  it('Active state', () => {
-    const wrapper = shallowMount(OkuToggle, {
-      propsData: {
-        defaultPressed: true,
-      },
-    })
-    expect(wrapper.attributes('aria-pressed')).toBe('true')
-  })
-
-  it('Inactive state', () => {
-    const wrapper = shallowMount(OkuToggle, {
-      propsData: {
-        defaultPressed: false,
-      },
-    })
-    expect(wrapper.attributes('aria-pressed')).toBe('false')
-  })
-
```

Comment:
Without deleting these tests, we should write a test with a new one. We should equate v-model and check 'aria-pressed'. Let's not delete the reverses.

Reply:
I directly set "pressed". How does it work?